### DISCUSSION TITLE
How can I run the project from source code?

### DISCUSSION THREAD

**Discussion by zjcscut** [consulo/consulo#633] [General] [answered]
- IDE build number: main branch
- Java version: JDK 22
- OS name and version:  macOS  Sequoia 15.1.1 (24B91)

I have been succeed to compile and package the project, how can I start it (or where is the main entry)?

<img width="1598" alt="image" src="https://github.com/user-attachments/assets/785d1bd4-357a-475b-b22b-4833ba88cc58">

**Comment by VISTALL** [comment, marked as answer]
Hello. Running Consulo from another IDE works but not fully supported. You need run maven from root dir with command

> consulo:run-desktop-awt-fork -pl consulo:consulo-sandbox-desktop-awt

(pull last changes from repository)

**Reply by zjcscut** [reply]
Got it, I will try it later.
